README.md

```diff
@@ -69,4 +69,26 @@ To run the tests for the codebase run the following command:
 
 poetry run pytest
 
-(please make sure you have run 'poetry install' beforehand to instal pytest)
\ No newline at end of file
+(please make sure you have run 'poetry install' beforehand to instal pytest)
+
+## Deploiyng the app via ansible 
+TO deploy the application via ansible copy the 'ansible' folder to the host node, update the inventory file (to include the control nodes you'd like to deploy to)and run the following command:
+ansible-playbook playbook.yaml -i inventory.yaml
+
+Please note you will need to have set up passwordless SSH access from the host to each of the managed nodes
```

Comment:
These instructions assumed you have completed exercise 4 (on Ansible). We haven't do this so this section should be removed (as it claims we have something we don't!)